recipes/airflow-with-celery/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name = "airflow-with-celery" %}
+{% set version = "1.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+build:
+  number: 0
+  skip: True  # [py3k or win]
+
+requirements:
+  run:
+    - airflow 1.8.0
+    - celery >=3.1.17
+    - flower >=0.7.3
+
+test:
+  commands:
+    - true
+
+about:
+  home: http://github.com/conda-forge/airflow-feedstock
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
where should I pickup the license from? There is no source code in this package.